<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaskEditorUI</class>
 <widget class="QDialog" name="MaskEditorUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/res/ball.ico</normaloff>:/res/ball.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Mask shape</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="rbWings">
        <property name="text">
         <string>Double Wings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbCircularDisk">
        <property name="text">
         <string>Circular Disk</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbRectangularDisk">
        <property name="text">
         <string>Rectangular Disk</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbDoubleWingWindow">
        <property name="text">
         <string>Double Wing Window</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbCircularWindow">
        <property name="text">
         <string>Circular Window</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbRectangularWindow">
        <property name="text">
         <string>Rectangular Window</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="4">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>152</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btnAdd">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="btnReset">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="btnClear">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
